otx/algorithms/segmentation/configs/base/data/selfsl/data_pipeline.py

```diff
@@ -0,0 +1,56 @@
+"""Data Pipeline for Self-SL model of Segmentation Task."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+
+__resize_target_size = 224
+__img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+
+__train_pipeline = [
+    dict(type="LoadImageFromFile"),
+    dict(type="LoadAnnotations"),
+    dict(
+        type="TwoCropTransform",
+        view0=[
+            dict(type="RandomResizedCrop", size=__resize_target_size),
+            dict(type="RandomFlip", prob=0.5, direction="horizontal"),
+            dict(
```

Comment:
I found that from ColorJitter to GaussianBlur each pipeline includes the steps to convert to PIL images. I think you can fix this conversion at once by putting [such pipeline methods](https://github.com/openvinotoolkit/model_preparation_algorithm/blob/8c17768043def5e7ef9023fcb5b9cc9b3cb3a987/mpa/modules/datasets/pipelines/torchvision2mmdet.py#L151) both in the start and end of pipelines. And I also wanna suggest to include probs arguments for the pipeline method itself rather than using ProbCompose to remove redundant format conversion.

Reply:
Thanks for the recommendation. I added that conversion pipeline to reduce redundant conversion and added probability in each module to avoid using `ProbCompose`, and updated the recipe.